tests/tensorflow/nightly/vision-retinanet.libsonnet

```diff
@@ -0,0 +1,113 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+local experimental = import '../experimental.libsonnet';
+local common = import 'common.libsonnet';
+local mixins = import 'templates/mixins.libsonnet';
+local timeouts = import 'templates/timeouts.libsonnet';
+local tpus = import 'templates/tpus.libsonnet';
+local utils = import 'templates/utils.libsonnet';
+
+{
+  local retinanet = common.TfVisionTest + {
+    modelName: 'vision-retinanet',
+    scriptConfig+: {
+      experiment: 'retinanet_resnetfpn_coco',
+    },
+    PreemptibleTpu:: {
```

Comment:
Remove this `PreemptibleTpu` block. It probably doesn't do what you want it to, and we want to control TPU preemptibility in common templates.